lib/gui/app.js

```diff
@@ -87,22 +91,29 @@ app.run((AnalyticsService, ErrorService, UpdateNotifierService, SelectionStateMo
   AnalyticsService.logEvent('Application start');
 
   const shouldCheckForUpdates = UpdateNotifierService.shouldCheckForUpdates();
-
-  if (!shouldCheckForUpdates || process.env.ETCHER_DISABLE_UPDATES) {
+  const currentVersion = packageJSON.version;
```

Comment:
Ahh, I didn't know `ManifestBindService` was part of the Angular stuff :)

`app-info` sounds good to me, but it also sounds like it ought to be a separate PR rather than part of this one ;-)